tests/conftest.py

```diff
@@ -1,4 +1,5 @@
 import pytest
+import asyncio
```

Comment:
This import should be above `import pytest`. You could use `isort` to sort the imports for you.